micromanager_gui/multid_widget.py

```diff
@@ -294,6 +289,94 @@ def set_multi_d_acq_dir(self):
         self.dir_lineEdit.setText(self.save_dir)
         self.parent_path = Path(self.save_dir)
 
+    def set_state(self, state: dict | MDASequence | str | Path) -> None:
```

Comment:
Similarly would be good to make `get_state` public

Reply:
Should it return a dict or an MDASequence?

Reply:
default to sequence with optional arg to make it as a dict?

Reply:
I'll do sequence.  I dislike parameters that change return types.   If someone wants a dict they can always do `get_state().dict()`

Reply:
made it public, added round-trip test